src/views/VenuesListWithFilter/FilterList/styles.module.css

```diff
@@ -0,0 +1,47 @@
+.filterList {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background: #FDFDFD;
+    border: 1px solid rgba(0, 0, 0, 0.1);
+    width: 282px;
```

Comment:
Do we want this width on a mobile version?

Reply:
Yes. Minimum size is 320px on Iphone SE, so it will look good on every resolution